index.html

```diff
@@ -418,6 +419,145 @@ <h2>Transferable MediaStreamTrack</h2>
       <p>The underlying source is supposed to be kept alive between the transfer and transfer-receiving steps, or as long as the data holder is alive.
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
+    <h3>MediaStreamTrack frame counters</h3>
+    <div>
+      <p>
+        Frame counters allow you to tell what the capture frame rate is, which
+        may be different from what is expected from settings - for instance, the
```

Comment:
Unclear who "you" is. Might also be specific about which settings.
```suggestion
        Frame counters allow applications to tell what the capture frame rate is, which
        may be lower than the target {{MediaTrackSettings/frameRate}}  - for instance, the
```
The prose also mentions "capture" which seems to assume camera only? (camera is mentioned in a for instance, which seems ok)

Reply:
The name itself `framesCaptured` implies this is only for video MST's returned by `getUserMedia`. Is that the intent?

If so then we have a `getStats` method that returns `{}` for other tracks, which seems fine I guess.

Reply:
This metric is not intended to be specific to GUM-tracks, it's just a frame counter. Do you prefer "framesProduced"? I would expect this metric to work for any track that has frames flowing

Reply:
Or just "frames"? They're meant to be frames not dropped, whether captured, produced through other means or from a cloned source

As a requirement it could be phrased as a counter that goes up for every non-dropped frame, measured as late in the pipeline as possible, but before the sink(s)